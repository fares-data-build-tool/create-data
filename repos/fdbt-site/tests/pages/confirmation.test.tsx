import * as React from 'react';
import { shallow } from 'enzyme';
import Confirmation from '../../src/pages/confirmation';

describe('pages', () => {
    describe('confirmation', () => {
        it('should render correctly', () => {
            const tree = shallow(<Confirmation operator="operator" service="service" />);
            expect(tree).toMatchSnapshot();
        });
    });
});
