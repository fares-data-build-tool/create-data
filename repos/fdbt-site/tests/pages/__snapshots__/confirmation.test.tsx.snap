// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages confirmation should render correctly 1`] = `
<Component
  description="Confirmation page of the Fares Data Build Tool"
  errors={Array []}
  title="Confirmation - Fares Data Build Tool"
>
  <CsrfForm
    action="/api/confirmation"
    csrfToken=""
    method="post"
  >
    <div
      className="govuk-grid-row"
    >
      <div
        className="govuk-grid-column-one-thirds"
      >
        <h1
          className="govuk-heading-l"
        >
          Check your answers before sending your fares information
        </h1>
        <ConfirmationTable
          confirmationElements={
            Array [
              Object {
                "header": "Fare Information",
                "innerElements": Array [
                  Object {
                    "content": "Flat Fare",
                    "href": "/fareType",
                    "name": "Faretype",
                  },
                  Object {
                    "content": "Group",
                    "href": "/passengerType",
                    "name": "Passenger Type",
                  },
                  Object {
                    "content": "Yes",
                    "href": "/timeRestrictions",
                    "name": "Time Restrictions",
                  },
                ],
              },
            ]
          }
        />
        <h2
          className="govuk-heading-m"
        >
          Now send your fares information
        </h2>
        <p
          className="govuk-body"
        >
          By submitting this notification you are confirming that, to the best of your knowledge, the details you are providing are correct.
        </p>
      </div>
      <input
        className="govuk-button"
        id="continue-button"
        type="submit"
        value="Continue"
      />
    </div>
  </CsrfForm>
</Component>
`;
